src/styles/grade.scss

```diff
@@ -0,0 +1,26 @@
+.grade {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 24px;
+  font-size: 10px;
+  line-height: 14px;
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+
+    &-star {
+      padding-right: 4px;
+      background-image: url('../images/star.svg');
+      width: 16px;
+      height: 16px;
+      background-repeat: no-repeat;
+
+      &:nth-child(-n + 4) {
+        background-image: url('../images/star-active.svg');
+      }
```

Comment:
The use of the `:nth-child(-n + 4)` selector assumes that there will always be exactly 4 active stars, which might not be flexible enough if the number of active stars needs to change dynamically. If the number of active stars is to be determined dynamically, consider using a class to indicate active stars instead of the `nth-child` pseudo-class.